i18n/zh/docusaurus-plugin-content-pages/case-studies/Kuiper/index.mdx

```diff
@@ -0,0 +1,170 @@
+---
+date: 2023-12-27
+title: Kuiper
+subTitle: 边缘流式数据处理平台
+description: 通过KubeEdge和Kuiper结合，提供灵活定义规则、告警和消息转发等能力，实现边缘流式数据处理。
+tags:
+  - 用户案例
```

Comment:
This is a solution

Reply:
Yes, the tag has been modified to solution.